### PR TITLE
Add User Management Doc

### PR DESCRIPTION
**Pull request by khorne3**
Moving this content over from ReadMe

### CODE REVIEW

File: admin/user-management.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+their password immediately after they log in.
+
+## Deleting a User
+
+To delete a user, go to **Manage** > **Users**. Find the user you want to delete
+and click the **Vertical Elipses** associated with that user.
+
+Click **Delete**.
+
+You'll be prompted to confirm this action. Click **Delete** to proceed.
```
Comment (vapurrmaid) [inline admin/user-management.md:36]: Should these be enumerated points?

File: admin/user-management.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "User Management"
+description: Learn how to add and delete Coder users.
+---
+
+Site managers can create and manage users via the User Management dashboard,
+which is accessible under **Manage** > **Users**.
```
Comment (vapurrmaid) [inline admin/user-management.md:7]: What do you think of the following:

```txt
[Site managers](link-to-roles) can create and manage users from the users page of the dashboard
which is accessible under **Manage** > **Users**.
```

Rationale:

- opportunity to link to rbac roles
- the capital casing of `User Manage` followed by lowercase dashboard feels a bit out of place, and we don't call it that in-app
- it's really a page within the dashboard, as opposed to a separate dashboard (current wording makes it feel like a separate app)

Not gonna die on any of these hills, but thought I'd suggest it while we're here

File: admin/user-management.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+title: "User Management"
+description: Learn how to add and delete Coder users.
```
Comment (vapurrmaid) [inline admin/user-management.md:3]:
```suggestion
description: Learn how to add, delete and manage Coder users.
```

File: admin/user-management.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+   Type if you would like to use your organization's OpenID Connect Identity
+   Provider.
+
+3. Finally, choose the **Organization** to which the user belongs (this will affect
+the images to which the user has access). Click **Create**.
+
+Coder will create the new user and display a Temporary Password. Provide this
+password to the user, which they can use along with their email to access their
+new account. For increased security, the new user will be prompted to change
+their password immediately after they log in.
```
Comment (vapurrmaid) [inline admin/user-management.md:28]: This is only true for `Built-In`

**Comment by khorne3** [comment]
@vapurrmaid All of your suggestions were great; thank you!

**Review by vapurrmaid — COMMENTED** [review]
Having trouble accessing the preview atm - but this _looks good_. I'll try to follow up shortly once I sort the Vercel access out

**Review by vapurrmaid — APPROVED** [review]

**Pull request merged**